pkg/daemon/ceph/multus/teardown.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package multus
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+)
+
+func Teardown() error {
```

Comment:
How critical is this action to tear down the multus interface? If the pod using the interface dies, it doesn't just go away automatically? We can't guarantee teardown will happen when a node goes down, so I wonder if we can just avoid this teardown completely.